test/bindings/nodejs/transit_available.test.js

```diff
@@ -0,0 +1,34 @@
+const test = require('tape');
+var config = require('./fixtures/basic_config');
+var Valhalla = require('../../../')(JSON.stringify(config));
+var valhalla = new Valhalla(JSON.stringify(config));
+
+test('transitAvailable: can return info for locations', function(assert) {
+  var transitAvailableRequest = '{"locations": [{"lon": -76.386018, "lat": 40.543564, "radius": 20},{"lon": -76.386040, "lat": 40.543939, "radius": 20}]}';
+  var transitAvailable = JSON.parse(valhalla.transitAvailable(transitAvailableRequest));
+  assert.ok(transitAvailable);
+  assert.equal(transitAvailable.length, 2, 'returns two availabilities, one for each point');
+  assert.equal(transitAvailable[0].istransit, false, 'no transit found');
+  assert.end();
+});
+
+test('transitAvailable: returns istransit false if bogus values passed in', function(assert) {
+  var badRequest = '{"locations": [{"lon": 500, "lat": 40.543564, "radius": 20},{"lon": 500, "lat": 40.543939, "radius": 20}]}';
+  var transitAvailable = JSON.parse(valhalla.transitAvailable(badRequest));
+  assert.ok(transitAvailable);
+  assert.equal(transitAvailable[0].istransit, false, 'no transit found');
+  assert.end();
+});
+
+test('transitAvailable: returns an error if request format is wrong', function(assert) {
+  var badRequest = '{"locations":[40.546115,-76.385076], [40.544232,"lon":-76.385752],"costing":"auto"}';
+  // TODO: we are not throwing a very useful error in this case - we should track this
```

Comment:
It's great we have these tests to document the behavior, but these inline TODOs are not actionable. I think we should remove the comments and instead write a ticket to follow up on throwing better errors.

Reply:
there is already a ticket - a member of the community mentioned a similar concern. I'll remove the TODOs, although I have noticed that this is a practice that is common in this codebase.

Reply:
imho these `TODO:`s aren't harmful. maybe a more appropriate label would be `NOTE:` to remove the idea that you have to do something at this point in the code. to me it reads like a note about expectations at this point. just my 2 cents, feel free to ignore